### PR TITLE
docs(features): external user grant

### PR DESCRIPTION
**Pull request by mffap**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/concepts/features/external-user-grant.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: External user grant
```
Comment (fforootd) [inline docs/docs/concepts/features/external-user-grant.md:2]:
```suggestion
title: External User Grant
```

Comment (fforootd) [reply docs/docs/concepts/features/external-user-grant.md:2]: I guess we use title case, right?

File: docs/docs/concepts/features/external-user-grant.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+ZITADEL's external user grant is a feature that allows you to grant access to projects within your organization to users from other organizations.
+This is useful in scenarios where you want to collaborate with external users without needing them to be part of your main organization.
```
Comment (fforootd) [inline docs/docs/concepts/features/external-user-grant.md:6]:
```suggestion
This is useful in scenarios where you want to collaborate with external users without needing them to be part of your organization.
```

Comment (fforootd) [reply docs/docs/concepts/features/external-user-grant.md:6]: I think we should get rid of "main" or elaborate it.

File: docs/docs/concepts/features/external-user-grant.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+By using external user grants, you can streamline collaboration with external users while maintaining control over access to your projects within ZITADEL.
+
+![](/img/concepts/features/external-user-grant.png)
+
+## Where to store users
```
Comment (fforootd) [inline docs/docs/concepts/features/external-user-grant.md:11]:
```suggestion
## Where to Store Users
```

File: docs/docs/concepts/features/external-user-grant.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### Consumer Identity Management (CIAM) / Business-to-Consumer (B2C)
+
+You might typically store all users in a single ZITADEL [organization](../structure/organizations) for managing customer accounts.
+We recommend creating a second organization for your own team that also contains all the projects and applications that will be [granted](../structure/granted_projects) to the first organization with the B2C customer accounts.
+Instead of duplicating user accounts for your team members in the B2C organization, you can create external user grants on the B2C organization.
+
+### Multitenancy / Business-to-Business (B2B)
```
Comment (fforootd) [inline docs/docs/concepts/features/external-user-grant.md:19]:
```suggestion
### Multi-Tenancy / Business-to-Business (B2B)
```

File: docs/docs/concepts/features/external-user-grant.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+ZITADEL allows you to create separate [organizations](../structure/organizations) for each of your business partner or tenant.
+There might be cases were users from one organization need access to projects from another organization.
+You can create an external user grant, that then allows the inviting organization to manage the roles for the external user.
```
Comment (fforootd) [inline docs/docs/concepts/features/external-user-grant.md:23]:
```suggestion
You can create an external user grant, that allows the inviting organization to manage the roles for the external user.
```

File: docs/docs/concepts/features/external-user-grant.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+## Project Grants vs. User Grants
+
+Project grants are used to delegate access management of an entire project to another organization.
```
Comment (fforootd) [inline docs/docs/concepts/features/external-user-grant.md:27]:
```suggestion
Project grants are used to delegate access management of an entire project (or specific roles of the project) to another organization.
```

File: docs/docs/concepts/features/external-user-grant.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+User grants provide a more granular approach, allowing specific users from external organizations to access your projects.
+
+## Alternative to multiple user accounts
```
Comment (fforootd) [inline docs/docs/concepts/features/external-user-grant.md:31]:
```suggestion
## Alternative to Multiple User Accounts
```

File: docs/docs/concepts/features/external-user-grant.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+A user account is always unique across a whole ZITADEL instance.
```
Comment (fforootd) [inline docs/docs/concepts/features/external-user-grant.md:33]:
```suggestion
A user account is always unique across a ZITADEL instance.
```

**Review by fforootd — COMMENTED** [review]
Some small improvements from my end

File: docs/docs/concepts/features/external-user-grant.md (hunk 1 above)
Comment (mffap) [reply docs/docs/concepts/features/external-user-grant.md:2]: No, we use lower case as per contributing guide :)

Comment (fforootd) [reply docs/docs/concepts/features/external-user-grant.md:2]: I guess you meant the google guidelines which use sentence case, right.

https://developers.google.com/style/capitalization

Comment (mffap) [reply docs/docs/concepts/features/external-user-grant.md:2]: Yes.
Side-note: title case is easy to achieve with css. The other way round not if we use own names (eg, ZITADEL, Organization, ...)

**Review by fforootd — APPROVED** [review]

**Review by dakshitha — APPROVED** [review]
Looks good to me.

**Pull request merged**